File: 프로그래머스/0/181894. 2의 영역/2의 영역.java
class Solution {
    public int[] solution(int[] arr) {
        int min = 999_999, max = 0;
        for(int i = 0 ; i < arr.length ; i++){
            if(arr[i] == 2) {
                if(i < min) min = i;
                if(i > max) max = i;
            }
        }
        if(min == 999_999) return new int[]{-1};
        int[] answer = new int[max-min+1];
        int num = 0;
        for(int i = min ; i <= max ; i++) {
            answer[num] = arr[i];
            num++;
        }
        return answer;
    }
}

File: 프로그래머스/0/181894. 2의 영역/2의 영역.py
def solution(arr):
    min, max = 111111, 0
    for i in range(len(arr)):
        if arr[i] == 2:
            if i < min:
                min = i
            if i > max:
                max = i
                
    if min == 111111 and max == 0:
        return [-1]
        
    return arr[min:max+1]
                    
        